web/pimcore/static6/js/pimcore/asset/listfolder.js

```diff
@@ -250,7 +256,71 @@ pimcore.asset.listfolder = Class.create({
 
         e.stopEvent();
         menu.showAt(e.getXY());
-    }
+    },
+
+    downloadSelectedZip: function () {
+
+        //create the ids array which contains chosen rows to export
+        ids = [];
+        var selectedRows = this.grid.getSelectionModel().getSelection();
+        for (var i = 0; i < selectedRows.length; i++) {
+            ids.push(selectedRows[i].data.id);
+        }
+        
+        Ext.Ajax.request({
```

Comment:
This code is 99,9% identical with https://github.com/pimcore/pimcore/blob/aea60550fae338e73bdb4435a0cc0b7ae370570e/web/pimcore/static6/js/pimcore/asset/folder.js#L316
We should reuse the existing code, eg. by passing the params as an argument of `pimcore.asset.folder.downloadZip`.

Reply:
Done! :)